src/components/runtime-common.js

```diff
@@ -39,10 +39,14 @@ export const RuntimeKicker = ({ runtime, refreshRuntimes, onNullRuntime }) => {
         await refreshRuntimes()
         setBusy(false)
         return
-      } else if (currentRuntime === undefined || status === 'Stopping') {
+      } else if (currentRuntime === undefined || status === 'Stopping' ) {
         await Utils.delay(500)
       } else if (currentRuntime === null) {
-        onNullRuntime()
+        await Utils.delay(1000)
```

Comment:
Is there a way you can poll for a certain condition to be true? Delays are notorious for causing flakiness.

Reply:
Not that I'm aware of. Since this condition will be true sometimes if there is no runtime, we can't just wait until a runtime is returned, we need a threshold condition like a wait to stop this from infinitely waiting until a runtime is returned. I'm open to suggestions though!

Reply:
Actually, I think we can just delete this block entirely. There's no real need for this component to ask the user the create a runtime if one doesn't exist. 

The relevant pages (the app and analysis launcher) display an intelligent message if the user ends up on it without a runtime. 

Furthermore, the corresponding operations ('open' in the launchers) also prompt the user to create an analysis if one doesn't exist. Finally, if a user finds themselves in this situation, they can also just create a runtime from the context bar.

Reply:
That would be awesome-- thanks for thinking through the scenarios @jdcanas!

Reply:
@jdcanas would removing the block I added and then passing `onNullRuntime: () => undefined` on [this line](https://github.com/DataBiosphere/terra-ui/blob/e786d1035ffea48f06529f708e5c47b39d1e3468/src/pages/workspaces/workspace/notebooks/NotebookLauncher.js#L82) be a better solution?

Reply:
I think `onNullRuntime` can also be removed entitrely